scrapscript.py

```diff
@@ -3448,6 +3469,20 @@ def listlength(obj: Object) -> Object:
 }
 
 
+PRELUDE = """
```

Comment:
Perhaps the prelude should be implemented as a dict sort of like stdlib? To allow multiple functions in the prelude?

Reply:
Or a list. Not sure. But not sure how we'd expand the prelude if it's just a string

Reply:
you can use `.` syntax

Reply:
Ok that's fine. Might prefer a list of strings and then we eval each and add them to the boot env..
But I trust whatever you think is best